config/utils.js

```diff
@@ -0,0 +1,44 @@
+const envOrThrow = (value) => {
+  const setValue = process.env[value]
+  if (!setValue) throw new Error(`Missing env var ${value}`)
+
+  return setValue
+}
+
+const envOr = (value, fallback) => {
+  const setValue = process.env[value]
+  if (!setValue) return fallback
+
+  return setValue
```

Comment:
Again personal style, this could be a single line function 🤔 
```
const envOr = (value, fallback) => process.env[value] || fallback
```

Reply:
😅 I was trying to write in _learner code_